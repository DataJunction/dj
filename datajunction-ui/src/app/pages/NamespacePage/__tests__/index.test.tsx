import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { NamespacePage } from '../index';

const renderer = createRenderer();

describe('<NamespacePage />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<NamespacePage />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
