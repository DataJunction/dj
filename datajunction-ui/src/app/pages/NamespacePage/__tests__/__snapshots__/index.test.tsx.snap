// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NamespacePage /> should render and match the snapshot 1`] = `
<div
  className="mid"
>
  <div
    className="card"
  >
    <div
      className="card-header"
    >
      <h2>
        Explore
      </h2>
      <div
        className="table-responsive"
      >
        <div
          className="sidebar"
        >
          <span
            style={
              Object {
                "color": "#95aac9",
                "fontSize": "0.8125rem",
                "fontWeight": "600",
                "padding": "1rem 1rem 1rem 0",
                "textTransform": "uppercase",
              }
            }
          >
            Namespaces
          </span>
        </div>
        <table
          className="card-table table"
        >
          <thead>
            <tr>
              <th>
                Name
              </th>
              <th>
                Type
              </th>
              <th>
                Status
              </th>
              <th>
                Mode
              </th>
              <th>
                Tags
              </th>
              <th>
                Last Updated
              </th>
            </tr>
          </thead>
          <tbody />
        </table>
      </div>
    </div>
  </div>
</div>
`;
